### DISCUSSION TITLE
Missing documentation/example for instant event and cron event

### DISCUSSION THREAD

**Discussion by dust63** [elsa-workflows/elsa-core#519] [Q&A]
Hi,

I'm trying to play with this activities but no success for me. Can you add docs for it and example.
I also remark it miss some documentation about Elsa activities with example and outcome definition and variable we can use in javascript or liquid.

**Comment by sfmskywalker** [comment]
For Elsa 1, I'm afraid the `Instant` event is actually completely broken and requires thorough refactorings that have been applied to Elsa 2.
The `Cron` event is slightly less broken, but still quite flaky. In Elsa 2, this has been completely fixed.

You can find an example for both Cron and Instant here: https://github.com/elsa-workflows/elsa-core/tree/feature/elsa-2.0/src/samples/worker/Elsa.Samples.Timers

Regarding missing documentation, you are right of course. This will be added as soon as Elsa 2 is in RC.

**Comment by sfmskywalker** [comment]
Which reminds me, I wrote a small article about the Cron event here: https://dev.to/sfmskywalker/scheduled-background-tasks-made-easy-with-elsa-workflows-3o6k

**Reply by dust63** [reply]
I think you can add reference on it on Elsa website ;). Thank you very much for your work. I planned to made file acitivity. 
How to propose new activity to be add on this marvelous project ?

**Reply by sfmskywalker** [reply]
Indeed. I need to add a blog post section or something similar there actually :)

If you would like to contribute a new activity of set of activities, the easiest way would be to fork the Elsa Core repo, create a new class library with your activities, and then submit a PR.

Thanks for the support!
